mitosheet/src/mito/Mito.tsx

```diff
@@ -129,6 +129,7 @@ export const Mito = (props: MitoProps): JSX.Element => {
         currOpenToolbarDropdown: undefined,
         toolbarDropdown: undefined,
         exportConfiguration: {exportType: 'csv'},
+        toolbarOpen: true,
```

Comment:
This feels like duplicated state. What if we just store the `currentTab` in the UIState, and then use that to check if the toolbar is open?